doc/techref/fonts.md

```diff
@@ -7,25 +11,105 @@ either `"Helvetica"` or `"0"`. For the special fonts "Symbol" (**12**) and
-| 6        | Times-Italic           | 23       | Helvetica-Narrow-Oblique     |
-| 7        | Times-BoldItalic       | 24       | Helvetica-Narrow-BoldOblique |
-| 8        | Courier                | 25       | NewCenturySchlbk-Roman       |
-| 9        | Courier-Bold           | 26       | NewCenturySchlbk-Italic      |
-| 10       | Courier-Oblique        | 27       | NewCenturySchlbk-Bold        |
-| 11       | Courier-BoldOblique    | 28       | NewCenturySchlbk-BoldItalic  |
-| 12       | Symbol                 | 29       | Palatino-Roman               |
-| 13       | AvantGarde-Book        | 30       | Palatino-Italic              |
-| 14       | AvantGarde-BookOblique | 31       | Palatino-Bold                |
-| 15       | AvantGarde-Demi        | 32       | Palatino-BoldItalic          |
-| 16       | AvantGarde-DemiOblique | 33       | ZapfChancery-MediumItalic    |
-|          |                        | 34       | ZapfDingbats                 |
-
-![Standard PostScript Fonts](https://docs.generic-mapping-tools.org/dev/_images/GMT_App_G.png){width="67.5%"}
+```{code-cell}
+---
+tags: [remove-input]
+---
+from IPython.display import display, Markdown
+
+fonts = [
+    "Helvetica",
+    "Helvetica-Bold",
+    "Helvetica-Oblique",
+    "Helvetica-BoldOblique",
+    "Times-Roman",
+    "Times-Bold",
+    "Times-Italic",
+    "Times-BoldItalic",
+    "Courier",
+    "Courier-Bold",
+    "Courier-Oblique",
+    "Courier-BoldOblique",
+    "Symbol",
+    "AvantGarde-Book",
+    "AvantGarde-BookOblique",
+    "AvantGarde-Demi",
+    "AvantGarde-DemiOblique",
+    "Bookman-Demi",
+    "Bookman-DemiItalic",
+    "Bookman-Light",
+    "Bookman-LightItalic",
+    "Helvetica-Narrow",
+    "Helvetica-Narrow-Bold",
+    "Helvetica-Narrow-Oblique",
+    "Helvetica-Narrow-BoldOblique",
+    "NewCenturySchlbk-Roman",
+    "NewCenturySchlbk-Italic",
+    "NewCenturySchlbk-Bold",
+    "NewCenturySchlbk-BoldItalic",
+    "Palatino-Roman",
+    "Palatino-Italic",
+    "Palatino-Bold",
+    "Palatino-BoldItalic",
+    "ZapfChancery-MediumItalic",
+    "ZapfDingbats",
+]
+
+text = "|Font No. | Font Name | Font No. | Font Name |\n"
+text += "|:---:|:---|:---:|:---|\n"
+for i in range(17):
+    j = i + 17
+    text += f"| {i} | {fonts[i]} | {j} | {fonts[j]} |\n"
+text += f"| | | 34 | {fonts[34]}\n"
+
+display(Markdown(text))
+```
+
+```{code-cell}
+---
+tags: [remove-input]
+---
+"""
+Script to generate visual samples of the fonts.
+"""
+import pygmt
+
+
+x1, x2, dx = 0.0, 7.0, 0.75
+
+fig = pygmt.Figure()
+# Draw the table
+fig.basemap(region=[-0.5, 14, -1.5, 18], projection="X14c/-10c", frame=0)
+fig.plot(x=[-0.5, 14], y=[-0.5, -0.5])
+for x in (0.5, 6.5, 7.5):
+    fig.plot(x=[x, x], y=[-1.5, 18])
+# Table header
+fig.text(
+    x=[x1, x1 + dx, x2, x2 + dx],
+    y=[-1.0] * 4,
+    text=["#", "Font Name"] * 2,
+    justify=["MC", "ML"] * 2,
+    font="Helvetica-Bold",
+)
+# Fonts
+for i, font in enumerate(fonts):
```

Comment:
Good to know that, it is possible to use variables defined in one code-cell in the following code-cells.